src/common.ts

```diff
@@ -11,7 +11,8 @@ export const pgPool = new PG.Pool();
  * toID - Turns anything into an ID (string with only lowercase alphanumeric characters)
  */
 export function toID(text: any): ID {
-	if (text && text.id) return text.id
+	// This is a premature optimization of sorts, but optional chaining in ID conversion is perhaps too slow.
+	if (text && text.id) return text.id; // eslint-disable-line @typescript-eslint/prefer-optional-chain
```

Comment:
toID is just PS's toID, if you think theres and improvement I'm open to discussion.

Reply:
yeah, i'm assuming that's why the optimization is there (also I guess for `toID(false)`).